state/src/stake/actions.rs

```diff
@@ -65,7 +65,7 @@ pub fn init_stake(
     stakeholders.save_to_state(state)?;
 
     for (address, deposit) in &genesis_candidates {
-        let balance = state.balance(address).unwrap_or_default();
+        let balance: u64 = Default::default();
```

Comment:
How about leaving some comments in the code to give a piece of information about the counterintuitive code?